tst/TurnConnectionFunctionalityTest.cpp

```diff
@@ -169,6 +169,7 @@ namespace com { namespace amazonaws { namespace kinesis { namespace video { name
         }
 
         EXPECT_TRUE(turnReady == TRUE);
+        EXPECT_EQ(STATUS_SUCCESS, turnConnectionStop(pTurnConnection));
```

Comment:
Is this just for function test coverage? Looks like it's not intensively enough to test turn connection stopping without sort of integration test.

Reply:
yes, this is just function test coverage.